modules/core/arrow-core/src/main/kotlin/arrow/core/Either.kt

```diff
@@ -247,6 +247,18 @@ fun <A, B, C> EitherOf<A, B>.flatMap(f: (B) -> Either<A, C>): Either<A, C> =
 inline fun <B> EitherOf<*, B>.getOrElse(crossinline default: () -> B): B =
   fix().fold({ default() }, ::identity)
 
+/**
+ * Returns the value from this [Either.Right] or null if this is a [Either.Left].
+ *
+ * Example:
+ * ```
+ * Right(12).orNull() // Result: 12
+ * Left(12).orNull()  // Result: null
+ * ```
+ */
+inline fun <B> EitherOf<*, B>.orNull(): B? =
```

Comment:
Why is it `*` here and `E` in the `Validated` one? If they're inconsistent across files we should fix it.

Reply:
I did notice that there were some inconsistencies between the different places I was changing. I tried to go with a "use the style of the code around you" approach, though, especially given that this is my first arrow PR, so I modeled each "orNull" after the "getOrElse" on the same class. Similarly, I modeled the tests after the corresponding "getOrElse" tests.